name: ffn_example_ds_config
workspace: DS_AT_PROTOTYPING
project: ffn_example
max_restarts: 0
resources:
  # slots_per_trial: 2
  slots_per_trial: 1
  shm_size: 17179869184 # 16 GiB. Was hitting "Unexpected bus error encountered in worker. This might be caused by insufficient shared memory (shm)."
  # max_slots: 8
  max_slots: 2
  # resource_pool: young # Grenoble specific; remove later.
searcher:
  name: custom
  metric: throughput
  smaller_is_better: False
hyperparameters:
  # dim: 512
  dim: 100
  layers: 2 # Total params will be layers * 1024 ** 2
  deepspeed_config: ds_config.json
entrypoint: python3 -m determined.launch.torch_distributed python3 script.py
